test/pem.js

```diff
@@ -625,6 +665,43 @@ exports['General Tests'] = {
           test.done();
       });
     },
+    'Check PKCS12 keystore': function(test) {
+      pem.createCertificate({
+          commonName: 'CA Certificate'
+      }, function (error, ca) {
+        test.ifError(error);
+
+        pem.createCertificate({
+            serviceKey: ca.serviceKey,
+            serviceCertificate: ca.certificate,
+            serial: Date.now(),
```

Comment:
Ok, thanks for checking this. Sorry for asking, but can you give that some priority in the next week(s)?